src/Aspire.Hosting.Azure.CognitiveServices/README.md

```diff
@@ -0,0 +1,44 @@
+# Aspire.Hosting.Azure.CognitiveServices library
+
+Provides extension methods and resource definitions for a .NET Aspire AppHost to configure Azure OpenAI.
+
+## Getting started
+
+### Prerequisites
+
+- Azure subscription - [create one for free](https://azure.microsoft.com/free/)
+- Azure OpenAI or OpenAI account - [create an Azure OpenAI Service resource](https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource)
+
+### Install the package
+
+In your AppHost project, install the .NET Aspire Azure Hosting Cognitive Services library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Azure.CognitiveServices
+```
+
+## Usage example
+
+Then, in the _Program.cs_ file of `AppHost`, add an Azure AI OpenAI service and consume the connection using the following methods:
+
+```csharp
+var openai = builder.AddAzureOpenAI("openai");
+
+var myService = builder.AddProject<Projects.MyService>()
+                       .WithReference(openai);
+```
+
+The `AddAzureOpenAI` method will read connection information from the AppHost's configuration (for example, from "user secrets") under the `ConnectionStrings:openai` config key. The `WithReference` method passes that connection information into a connection string named `openai` in the `MyService` project. In the _Program.cs_ file of `MyService`, the connection can be consumed using:
+
+```csharp
+builder.AddAzureOpenAIClient("openai");
+```
```

Comment:
One thing we do in the Components READMEs is to reference which `Aspire.Hosting.X` packages you need to reference to do the hosting side of the changes.

Maybe we could do the same thing here the other way?

Reply:
Yeah good point -- i basically extracted the 'app host extension' sections from the component ones here to be consistent, but now that is lacking what about a simpler thing here and link to the components -- that way the better instructions are there with more context?

Reply:
What do you think of this @eerhardt:
![image](https://github.com/dotnet/aspire/assets/4821/274e443b-606c-4dfe-87d7-7b1ea8dc697c)

Rather than repeating the install instructions, etc. --  just link to that package

Reply:
👍 I like it.